devtools/highlight.py

```diff
@@ -0,0 +1,77 @@
+import sys
```

Comment:
could you move all this code into `utils.py`

Reply:
Sure .. will rename `highlight.py` to `util.py`

Reply:
Done